edgelet/iotedge/src/bundle.rs

```diff
@@ -0,0 +1,233 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+use chrono::{DateTime, Local, NaiveDateTime, Utc};
```

Comment:
Order of imports is:

- libstd `use std::`
- 3rd party crates `use chrono::`
- Our crates `use edgelet_core::`
- Current crate `use crate::`
- Super imports `use super::`

Each group is internally sorted alphabetically and separated by an empty line.